src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -335,9 +335,17 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			fluid.name = modElement.getName();
 			fluid.textureFlowing = "test";
 			fluid.textureStill = "test2";
+			fluid.canMultiply = _true;
+			fluid.flowRate = 8;
+			fluid.levelDecrease = 2;
+			fluid.slopeFindDistance = 3;
+			fluid.spawnParticles = !_true;
+			fluid.dripParticle = new Particle(modElement.getWorkspace(),

```

Comment:
Consider adding test option if user does not select one

Reply:
Not sure what you mean by this, the test for the drip particle is based on the tests of other elements

Reply:
As dripParticle is optional, you could add check emptyLists ? empty variant : random particle, just idea to cover more test cases

Reply:
I don't think it's actually possible to pick null, the drip particle is toggled with a checkbox (`spawnParticles`) like the particles of blocks

Reply:
Ok, I see. But is it ok that there is no default value specified for existing fluids? I mean if it works then it is ok.

Reply:
From what I've seen it doesn't cause any issue with existing fluids